src/ucp/wireup/wireup_cm.c

```diff
@@ -312,20 +341,16 @@ ucp_cm_ep_priv_data_pack(ucp_ep_h ep, const ucp_tl_bitmap_t *tl_bitmap,
         goto err;
     }
 
-    sa_data_length = ucp_cm_priv_data_length(ucp_addr_size, sa_data_version);
-    cm_idx         = ep->ext->cm_idx;
-    if (worker->cms[cm_idx].attr.max_conn_priv < sa_data_length) {
-        ucs_log(log_level,
-                "CM private data buffer is too small to pack UCP endpoint"
-                " info, ep %p service data version %u, size %lu, address "
-                "length %lu, cm %p max_conn_priv %lu", ep, sa_data_version,
-                sa_data_length - ucp_addr_size, ucp_addr_size,
-                worker->cms[cm_idx].cm, worker->cms[cm_idx].attr.max_conn_priv);
-        status = UCS_ERR_BUFFER_TOO_SMALL;
+    status = ucp_wireup_cm_priv_space_check(worker, ep->ext->cm_idx,
+                                            ucp_addr_size, sa_data_version,
+                                            UCS_LOG_LEVEL_ERROR);
+    if (status != UCS_OK) {
+        ucs_error("CM private data buffer is not enough to pack ep %p", ep);
```

Comment:
why needed? ucp_wireup_cm_priv_space_check would already print the error

Reply:
I'll remove this ```error message log```.  Previously, this log is used to print ```ep pointer``` value because that ```ucp_wireup_cm_priv_space_check``` doesn't not output ```ep pointer```.